cdhweb/pages/blocks/accordion_block.py

```diff
@@ -64,7 +64,7 @@ class AccordionTitles(models.TextChoices):
         CODE = "code", "Code"
         DATA = "data", "Data"
         DOCUMENTATION = "documentation", "Documentation"
-        ANALYSIS = "analysis", "Analysis"
+        PUBLICATIONS_AND_PRESENTATIONS = "publications_and_presentations", "Publications and Presentations"
```

Comment:
Do we know what their casing preference is? 
I've been erring on the side of sentence case (e.g. "Publications and presentations") not title case (e.g. "Publications and presentations"). I think sentence case is becoming more common these days.

Reply:
Looking at the strings we have below this one (`RELATED_COURSES`, `PROJECT_PEER_REVIEW`) it looks like they're all Title Case, and IIRC, there's a style-guide difference -- For what it's worth,[ APA recommends title-case for this situation](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)

Reply:
So is this OK to be left as it is? @haydngreatnews @liamjohnston

Reply:
Yep, good by me